### ISSUE TITLE
Warning: Error in FUN: Expected an object with class 'shiny.tag'.

### ISSUE THREAD

**Issue by olb314159** [Tychobra/polishedpayments#10]
Hi,

I appreciate the work you have done with polished payments. 

I installed the most recent version a few minutes ago. An error occured when attempting to go to the billing page after running a shiny app successfully.  

Error from the console:

```
<a id="account-go_to_shiny_app" href="#" class="action-button" style="margin-left: -15px; margin-top: -7.5px; color: #1a8dc7; font-size: 18px;">Go to Analytics</a>
Warning: Error in FUN: Expected an object with class 'shiny.tag'.
  69: ui
```

Thanks in advance for any assistance you can provide.

**Comment by socioskop** [comment]
You need to redefine the left_menu in line 36+ in the app_module.R file to a tags$li object and specify class as 'dropdown':

left_menu = tags$li(class="dropdown",
                        shiny::actionLink(
                          ns("go_to_shiny_app"),
                          paste0("Go to ", app_name),
                          style = "margin-left: -15px; margin-top: -7.5px; color: #1a8dc7; font-size: 18px;"
                        )

**Comment by merlinoa** [comment]
This has finally been fixed: 1dfefe520bd9c016de6c0a3919e582fe3daf54b4

Thanks for providing the fix @socioskop !
